qt/TorrentFilter.cc

```diff
@@ -15,6 +15,8 @@
 #include "TorrentModel.h"
 #include "Utils.h"
 
+const QString TorrentFilter::TRACKERLESS_FILTER = tr("(trackerless)");
```

Comment:
This is wrong. Static construction happens before execution enters the `main()` function, which means translators aren't yet set up and we'll always get an English phrase here. Also note that all the code now has `const` on the right of the type, not on the left.